src/witness/yamlWitness.ml

```diff
@@ -553,13 +589,28 @@ struct
           None
       in
 
+      let validate_location_invariant (location_invariant: YamlWitnessType.LocationInvariant.t) =
+        let loc = loc_of_location location_invariant.location in
+        let inv = location_invariant.location_invariant.string in
+        let entry_certificate = Entry.loop_invariant_certificate in (* TODO: wrong *)
```

Comment:
What is meant by `TODO: wrong` here?

Reply:
It's wrong in the sense that there's no `location_invariant_certificate`, nor any general invariant certificate for the time being, so this is the closest thing matching old behavior. Since we don't really care about the certificates, I think it's fine, but I can explain this also in the TODO.

Reply:
Yes, i think that might be helpful for others like me who are not intimately familiar with witnesses.